static/bidder-info/krushmedia.yaml

```diff
@@ -14,5 +14,8 @@ capabilities:
       - native
 userSync:
   redirect:
-    url: "https://cs.krushmedia.com/4e4abdd5ecc661643458a730b1aa927d.gif?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
+    url: "https://cs.krushmedia.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
Confirmed redirect is working
<img width="1001" alt="Screenshot 2024-11-04 at 5 01 02 PM" src="https://github.com/user-attachments/assets/5ba3052b-9849-4c7d-a42e-74145dcb3f1e">